docs/data-ai/train/capture-images.md

```diff
@@ -0,0 +1,283 @@
+---
+linkTitle: "Capture images for training"
+title: "Capture images for training"
+weight: 20
+layout: "docs"
+type: "docs"
+description: "Capture images that you can use to train a machine learning model."
+---
+
+## Prerequisites
+
+{{% expand "A machine connected to Viam" %}}
+
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or similar [camera component](/operate/reference/components/camera/).
+
+{{% /expand%}}
+
+{{< alert title="Tip" color="tip" >}}
+
+For the best results, use the same camera for both training data capture and production deployment.
+
+{{< /alert >}}
+
+## Capture individual images
+
+{{< tabs >}}
+{{% tab name="Web UI" %}}
+
+You can add images to a dataset directly from a camera or vision component feed in the machine's **CONTROL** or **CONFIGURATION** tabs.
+
+To add an image directly to a dataset from a visual feed, complete the following steps:
+
+1. Open the **TEST** panel of any camera or vision service component to view a feed of images from the camera.
+1. Click the button marked with the camera icon to save the currently displayed image to a dataset:
+   {{< imgproc src="/components/camera/add_image_to_dataset_button.png" alt="A button marked with the outline of a camera, emphasized in red" resize="800x" style="width:500px" class="imgzoom" >}}
+1. Select an existing dataset.
+1. Click **Add** to add the image to the selected dataset.
+1. When you see a success notification that reads "Saved image to dataset", you have successfully added the image to the dataset.
+
+To view images added to your dataset, go to the **DATA** page, open the [**DATASETS** tab](https://app.viam.com/data/datasets), then select your dataset.
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+To capture an image and add it to your **DATA** page, fetch an image from your camera through your machine.
+Pass that image and an appropriate set of metadata to [`data_client.binary_data_capture_upload`](/dev/reference/apis/data-client/#binarydatacaptureupload):
+
+```python
+CAMERA_NAME = "<camera-name>"
+MACHINE_ADDRESS = "<machine-address.viam.cloud>"
+
+dial_options = DialOptions(
+    credentials=Credentials(
+        type="api-key",
+        payload=API_KEY,
+    ),
+    auth_entity=API_KEY_ID,
+)
+
+robot_opts = RobotClient.Options.with_api_key(
+    api_key=API_KEY,
+    api_key_id=API_KEY_ID
+)
+
+viam_client = await ViamClient.create_from_dial_options(dial_options)
+data_client = viam_client.data_client
+
+robot_client = await RobotClient.at_address(ROBOT_ADDRESS, robot_opts)
+camera = Camera.from_robot(robot_client, CAMERA_NAME)
+
+# Capture image
+image_frame = await camera.get_image()
+
+# Upload data
+file_id = await data_client.binary_data_capture_upload(
+    part_id=PART_ID,
+    component_type="camera",
+    component_name=CAMERA_NAME,
+    method_name="GetImages",
```

Comment:
Wait why are all of these saying the method name is GetImages instead of GetImage? Shouldn't it match the method you're using on line 78? Same applies to other language samples.

Reply:
Good point, I must have forgotten to update this part since it's declared in multiple places. Thanks!

```suggestion
    method_name="GetImage",
```